x-pack/legacy/plugins/reporting/server/plugin.ts

```diff
@@ -76,8 +86,8 @@ export function reportingPluginFactory(
         exportTypesRegistry
       );
 
-      const logger = LevelLogger.createForServer(__LEGACY, [PLUGIN_ID]);
-      const browserDriverFactory = await createBrowserDriverFactory(__LEGACY);
+      const logger = new LevelLogger(this.initializerContext.logger.get('reporting'));
+      const browserDriverFactory = await createBrowserDriverFactory(__LEGACY, logger);
```

Comment:
NOTE: the legacy `initializerContext.logger.get` returns a logger factory with empty context.

You have to be aware what once migrated to NP and using the actual NP plugin's `initializerContext`, this will be prefixed with the `plugins` context, meaning that this.initializerContext.logger.get('reporting') will returns the `plugins.reporting`-prefixed factory instead of just `reporting` as of now.